slack-base/src/main/java/com/hubspot/slack/client/methods/SlackMethods.java

```diff
@@ -618,7 +618,12 @@ public enum SlackMethods implements SlackMethod {
   views_open(MethodWriteMode.WRITE, RateLimitingTiers.TIER_4, JsonStatus.ACCEPTS_JSON),
   views_update(MethodWriteMode.WRITE, RateLimitingTiers.TIER_4, JsonStatus.ACCEPTS_JSON),
   views_push(MethodWriteMode.WRITE, RateLimitingTiers.TIER_4, JsonStatus.ACCEPTS_JSON),
-  views_publish(MethodWriteMode.WRITE, RateLimitingTiers.TIER_4, JsonStatus.ACCEPTS_JSON);
+  views_publish(MethodWriteMode.WRITE, RateLimitingTiers.TIER_4, JsonStatus.ACCEPTS_JSON),
+  files_getUploadURLExternal(
```

Comment:
should this be sorted? or would that cause a breaking enum ordinal change?

Reply:
Sorted


> breaking enum ordinal change

I do not think it is a problem here